qutebrowser/completion/completionwidget.py

```diff
@@ -192,10 +192,13 @@ def _next_prev_item(self, prev):
         Args:
             prev: True for prev item, False for next one.
         """
-        idx = self._next_idx(prev)
-        qtutils.ensure_valid(idx)
```

Comment:
Ah, no, seems like the test added to `test_completion_item_next_prev` catches it - my bad! I'll fix up the nitpicks and merge.